tools/serverpod_cli/lib/src/generator/dart/library_generators/class_generators/repository_classes.dart

```diff
@@ -915,8 +915,17 @@ class BuildRepositoryClass {
                 subDirParts: classDefinition.subDirParts,
                 config: config,
               );
-          })
+          }),
         ])
+        ..optionalParameters.add(
```

Comment:
Comment when reviewing: Would have been great to include the unit tests for this change in this commit as well (I'm reviewing commit by commit).

Reply:
Can absolutely do that in the future 👍